tests/functional_tests/functional_tests_rpc.py

```diff
@@ -82,6 +82,12 @@
   outputs.append(open(FUNCTIONAL_TESTS_DIRECTORY + '/wallet' + str(i) + '.log', 'a+'))
   ports.append(18090+i)
 
+# 1 offline wallet
+command_lines.append([str(18090+N_WALLETS) if x == "wallet_port" else x for x in wallet_base])
+command_lines[-1] += ["--offline"]
+outputs.append(open(FUNCTIONAL_TESTS_DIRECTORY + '/wallet' + str(N_WALLETS) + '.log', 'a+'))
+ports.append(18090+N_WALLETS)
+
```

Comment:
Any chance this cam be moved back to the loop since it's very similar ? It's just a if i == N_WALLETS-1:  command_lines[i] += "--offline"; AFAICT.